bundles/org.openhab.binding.insteon/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,444 @@
+
+	<channel-type id="keypadButtonH">
+		<item-type>Switch</item-type>
+		<label>Keypad Button H</label>
+	</channel-type>
+
+	<channel-type id="lastHeardFrom">
+		<item-type>DateTime</item-type>
+		<label>Last Heard From</label>
+	</channel-type>
+
+	<channel-type id="ledBrightness">
+		<item-type>Number</item-type>
+		<label>LED brightness</label>
+	</channel-type>
+
+	<channel-type id="lightDimmer">
+		<item-type>Dimmer</item-type>
+		<label>light Dimmer</label>
+	</channel-type>
+
+	<channel-type id="loadDimmer">
+		<item-type>Dimmer</item-type>
+		<label>Load Dimmer</label>
+	</channel-type>
+
+	<channel-type id="loadSwitch">
+		<item-type>Switch</item-type>
+		<label>Load Switch</label>
+	</channel-type>
+
+	<channel-type id="loadSwitchFastOnOff">
+		<item-type>Switch</item-type>
+		<label>Load Switch Fast On/Off</label>
+	</channel-type>
+
+	<channel-type id="loadSwitchManualChange">
+		<item-type>Number</item-type>
+		<label>Load Switch Manual Change</label>
+	</channel-type>
+
+	<channel-type id="manualChange">
+		<item-type>Number</item-type>
+		<label>Manual Change</label>
+	</channel-type>
+
+	<channel-type id="manualChangeButtonA">
+		<item-type>Number</item-type>
+		<label>Manual Change Button A</label>
+	</channel-type>
+
+	<channel-type id="manualChangeButtonB">
+		<item-type>Number</item-type>
+		<label>Manual Change Button B</label>
+	</channel-type>
+
+	<channel-type id="manualChangeButtonC">
+		<item-type>Number</item-type>
+		<label>Manual Change Button C</label>
+	</channel-type>
+
+	<channel-type id="manualChangeButtonD">
+		<item-type>Number</item-type>
+		<label>Manual Change Button D</label>
+	</channel-type>
+
+	<channel-type id="meterKWH">
+		<item-type>Number</item-type>
+		<label>Meter KWH</label>
+	</channel-type>
+
+	<channel-type id="meterReset">
+		<item-type>Switch</item-type>
+		<label>Meter Reset</label>
+	</channel-type>
+
+	<channel-type id="meterUpdate">
+		<item-type>Switch</item-type>
+		<label>Meter Update</label>
+	</channel-type>
+
+	<channel-type id="meterWatts">
+		<item-type>Number</item-type>
+		<label>Meter Watts</label>
+	</channel-type>
+
+	<channel-type id="notification">
+		<item-type>Number</item-type>
+		<label>Notification</label>
+	</channel-type>
+
+	<channel-type id="onLevel">
+		<item-type>Number</item-type>
+		<label>On Level</label>
+	</channel-type>
+
+	<channel-type id="rampDimmer">
+		<item-type>Dimmer</item-type>
+		<label>Ramp Dimmer</label>
+	</channel-type>
+
+	<channel-type id="rampRate">
+		<item-type>Number</item-type>
+		<label>Ramp Rate</label>
+	</channel-type>
+
+	<channel-type id="stage1Duration">
+		<item-type>Number</item-type>
+		<label>Stage 1 Duration</label>
+	</channel-type>
+
+	<channel-type id="switch">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+	</channel-type>
+
+	<channel-type id="systemMode">
+		<item-type>Number</item-type>
+		<label>System Mode</label>
+	</channel-type>
+
+	<channel-type id="tempCelsius">
+		<item-type>Number</item-type>
```

Comment:
This is better modeled as a UoM (QuantityType) type channel the there is no need for having a separate channel for Celsius and Fahrenheit.
```suggestion
		<item-type>Number:Temperature</item-type>
```

Reply:
I will update to use number:Temperature for both channels.  I left both units of measure since that is how the original binding was written. Since I don't have devices that I can test a single channel, I don't want to change since I can't easily validate

Reply:
Have a try nonetheless. We must not have separate celsius and fahrenheit channels in openHAB 2. So please merge them into a single "temp" channel. If you cannot test, just drop the Fahrenheit one and add Celsius as a unit when updating the channel state.

Reply:
done, implemented a single temperature channel. Hopefully I didn't break anything :)